content/02.introduction.md

```diff
@@ -1,31 +1,33 @@
+Researchers found a gap between male and female subjects or sources, with independent studies finding that between 17-40% of total subjects were female across multiple general-interest printed news outlets between 1985 and 2015 [@doi:10.1177/0003122415596999; @doi:10.1080/1461670X.2013.834149; @https://whomakesthenews.org/gmmp-2015-reports/].
+One study found 27-35% of total subjects in international science and health related news were female between 1995 and 2015, and 46% in print, radio, and television in the United States in 2015 [@https://whomakesthenews.org/gmmp-2015-reports/].
+It should be noted that scientific news coverage is confounded by the existing differences in gender and racial demographics within the scientific field [@https://ncses.nsf.gov/wmpd; @doi:10.1038/s41590-019-0470-6].
+However, we are interested in quantifying the additional level of bias beyond observed demographic differences in the scientific field, using authorship as an estimate for the existing demographic differences. 
+This is similar to other studies that have quantified gender or racial disparities in science as observed in citation [@doi:10.1177/2378023117738903; @doi:10.1038/504211a] and funding rates [@doi:10.1016/j.cell.2021.01.011; @doi:10.1126/sciadv.aaz4868; @doi:10.1126/sciadv.aaw7238; @doi:10.1126/science.1165878; @doi:10.1126/science.1196783].
+<!-- CUT and publication rates [@doi:10.1007/s11192-012-0893-4; @doi:10.1073/pnas.1915378117; @doi:10.1257/pandp.20181111]. -->
  
-Therefore, it is crucial to ensure that science coverage does not solely focus on a few well-known scientists but expands our shared view of an expert scientist.
-One may believe that science coverage would simply reflect the most current and groundbreaking findings. 
-Still, there are many ways gender, racial, or regional biases can unknowingly seep into coverage.
-In researching a story, a journalist will typically interview multiple scientists for their opinion, potentially asking for additional sources, allowing individual unconscious biases to skew scientific coverage broadly.
+<!-- CUT It is crucial to ensure that science news coverage does not solely focus on a few well-known scientists, but expands our shared view of an expert scientist. -->
+Even if science news coverage simply reflects the current scientific events and new findings, there are many gender, racial, or regional biases that can unknowingly seep into coverage.
+In researching a story, a journalist will typically interview multiple sources for their opinion, potentially asking for additional sources, thus allowing individual unconscious biases to skew scientific coverage broadly.
```

Comment:
```suggestion
In researching a story, a journalist will typically interview multiple sources for their opinion, potentially asking for additional sources, thus allowing individual unconscious biases at any point along the interview chain to skew scientific coverage broadly.
```
Not strong feeling at this, but I think this is what you are referring to more specifically. "at any point" might be removeable - not sure.